server/templates/ip_filters_changed.mjml

```diff
@@ -4,32 +4,42 @@
             <mj-include path="./partials/header.mjml" />
             <mj-section padding="0px">
                 <mj-column>
-                    <mj-image src="{{.Props.SiteURL}}/static/images/forgot_password_illustration.png" width="312px"
+                    <mj-image src="{{.Props.PortalURL}}/static/images/forgot_password_illustration.png" width="312px"
                         padding="0px" />
                 </mj-column>
             </mj-section>
             <mj-section padding="40px 0px 40px 0px">
                 <mj-column>
                     <mj-text padding-bottom="9px" css-class="footerTitle" padding="0px">
-                            {{.Props.TroubleAccessingTitle}}
+                        {{.Props.TroubleAccessingTitle}}
                     </mj-text>
                     <mj-raw>{{if .Props.ActorEmail}}</mj-raw>
-                    <mj-button padding-top="0px" padding-bottom="1px" font-size="14px" line-height="20px" background-color="transparent" color="#1C58D9" href="mailto:{{.Props.ActorEmail}}">
+                    <mj-button padding-top="0px" padding-bottom="1px" font-size="14px" line-height="20px"
+                        background-color="transparent" color="#1C58D9" href="mailto:{{.Props.ActorEmail}}">
                         {{.Props.SendAnEmailTo}}
                     </mj-button>
-                    <mj-divider padding="0" css-class="divider"  width="313px" border-width="1px" border-color="#3F4350"/>
+                    <mj-divider padding="0" css-class="divider" width="313px" border-width="1px"
+                        border-color="#3F4350" />
                     <mj-raw>{{end}}</mj-raw>
                     <mj-raw>{{ if .Props.LogInToCustomerPortal}}</mj-raw>
-                    <mj-button padding-top="6px" padding-bottom="1px" font-size="14px" line-height="20px"  background-color="transparent" color="#1C58D9" href="{{.Props.PortalURL}}/console/cloud/ip-filtering">
+                    <mj-button padding-top="6px" padding-bottom="1px" font-size="14px" line-height="20px"
+                        background-color="transparent" color="#1C58D9"
+                        href="{{.Props.PortalURL}}/console/cloud/ip-filtering">
                         {{.Props.LogInToCustomerPortal}}
                     </mj-button>
-                    <mj-divider padding="0px" css-class="divider" width="313px" border-width="1px" border-color="#3F4350"/>
+                    <mj-divider padding="0px" css-class="divider" width="313px" border-width="1px"
+                        border-color="#3F4350" />
                     <mj-raw>{{end}}</mj-raw>
-                    <mj-button padding-top="6px" font-size="14px" line-height="20px"  background-color="transparent" color="#1C58D9" href="mailto:{{.Props.SupportEmail}}">
+                    <mj-button padding-top="6px" font-size="14px" line-height="20px" background-color="transparent"
+                        color="#1C58D9" href="mailto:{{.Props.SupportEmail}}">
```

Comment:
These  changes look unintended (or half through). I would expect the each property in a different line, or everything in the same line, not just "arbitrarily" divided in the middle.

Reply:
Fixed!